src/marketing/create-new-rec.md

```diff
@@ -4,27 +4,39 @@ tag: product-recommendations
 conditions: Default.EE Only
 ---
 
-When you create a recommendation, you are creating what is called a *recommendation unit*. A recommendation unit has a title, such as "Recommended for you", "People also like", and so on. You can configure which pages should display the recommendation unit, where on the page you want the recommendation unit to appear, and which recommendation type you want to use, such as "most viewed", "most purchased", and so on. After you create and activate the recommendation unit, Magento will begin [collecting metrics]({% link marketing/recommendation-metrics.md %}) that measure data such as if a unit was rendered on a page, if it was viewed, if it was clicked by a user, and so on. The [Product Recommendations]({% link marketing/product-recommendations.md %}#dashboard) dashboard displays these metrics for each recommendation unit to help you make informed business decisions.
+When you create a recommendation, you are creating what is called a *recommendation unit*:
+
+   ![Recommendation unit]({% link images/images-ee/p-rex-unit.png %}){: .zoom}
+
+The products inside the above recommendation unit are called *recommendation items*. When you create a new recommendation, you configure:
+
+-  Which pages should display the recommendation unit
+
+-  Where on the page you want the recommendation unit to appear
+
+-  Which recommendation type you want to use, such as "most viewed", "most purchased", and so on
+
+After you create and activate the recommendation unit, Magento will begin [collecting metrics]({% link marketing/recommendation-metrics.md %}) that measure data such as if a unit was rendered on a page, if it was viewed, if it was clicked by a user, and so on. The [Product Recommendations]({% link marketing/product-recommendations.md %}#dashboard) dashboard displays these metrics for each recommendation unit to help you make informed business decisions.
 
 1. On the _Admin_ sidebar, go to **Marketing** > _Promotions_ > **Product Recommendations** to display the **Product Recommendations** dashboard.
 
 1. Specify the [Store View]({% link configuration/scope-change.md %}) where you want the recommendations to display.
 
 1. Click the **Create Recommendation** button.
 
-1. In the **Name your Recommendation** section, enter a name that uniquely identifies the recommendation. This is used for internal tracking only. Your users will not see this name.
+1. In the **Name your Recommendation** section, enter a name that uniquely identifies the recommendation. This is used for internal tracking only. Your shoppers will not see this name.
```

Comment:
remember, shoppers can still see this name in the browser dev console via network traffic, so we need to be careful here. Still need to make this change in the Admin UI... one option was "this name is not rendered on storefront" which may not be the best because it hinges on the definition of what "rendered" means

Reply:
I'll reword this.

Reply:
I'm going to remove the "Your shoppers will not see this name." sentence in favor of expanding what "This is used for internal tracking only." means.